tests/data/hyperboloid_data.py

```diff
@@ -23,26 +23,24 @@ class HyperboloidTestData(_LevelSetTestData):
 
     Space = Hyperboloid
 
+    def intrinsic_after_extrinsic_test_data(self):
+        return self.generate_tests([])
```

Comment:
Is that generating empty tests? Will it be confusing, as in: we think the tests pass but in fact nothing is being tested? Should we use the skip flags instead. Feel free to ignore this comment if you plan in addressing this in the upcoming test PR.

Reply:
This one shouldn't have been removed. I've added back the data. On the other hand, `extrinsic_after_intrinsic` needs to be skipped in this way because it assumes space can receive `default_point_type="intrinsic"`, which is not the case. Notice that data is still created when we skip with "skip_" flag, so I really need to skip it this way.

I'm addressing this in the new tests, but my first thought is that we need to rethink these methods. The reason is that I find counter-intuitive to have methods in a class that can receive point shapes the class is not supposed to handle, and this will always happen because:

* if we have an intrinsic representation, `extrinsic_to_intrinsic_coords` will receive an "unexpected" point type.
* if we have an extrinsic representation, `intrinsic_to_extrinsic_coords` will receive an "unexpected" point type.

This relates with the discussion in the last meeting regarding have this "PointRepresentationTransform" / "CoordsTransform" object.